crowbar_framework/app/controllers/installer/upgrades_controller.rb

```diff
@@ -226,11 +233,11 @@ def save_and_restore
       return false unless @backup.save
       if Crowbar::Backup::Restore.restore_steps_path.exist?
         flash[:info] = t("installer.upgrades.restore.multiple_restore")
-        true
       else
         Crowbar::Backup::Restore.purge
         @backup.restore(background: true, from_upgrade: true)
       end
+      true
```

Comment:
no we haven't yet, there is no specific return value as it just opens a thread (not even sure what it returns)

we just respect a false return value if the backup can't be saved